Updating to Laravel 5

oriceon opened the pull request.

oriceon committed:
Update to Laravel 5

oriceon committed:
Escape Toastr javascript render

piperone commented:
A Laravel 5 branch of this would be most welcome. If this PR works with L5, I'd say go for it! :+1:

oriceon committed:
Update README.md

oriceon committed:
Update composer.json

oriceon committed:
Update composer.json

oriceon committed:
Update README.md

oriceon committed:
Fix composer require package name

carlosrgzm committed:
Update Toastr.php

Remove htmlentites from the message to allow links in the notification's messages

carlosrgzm committed:
Changed package name

Changed package name to allow upload it to packagist

carlosrgzm committed:
Update config.php

carlosrgzm committed:
Update config.php

oriceon committed:
Merge pull request #1 from carlosrgzm/master

Allow html in text messages, for creating for instance links

oriceon committed:
Update composer.json

jozefbalun committed:
Support for laravel 5.4

oriceon committed:
Merge pull request #2 from jozefbalun/patch-1

Support for laravel 5.4

jozefbalun committed:
laravel 5.4 support - update

this should be better

oriceon committed:
Merge pull request #3 from jozefbalun/patch-2

laravel 5.4 support - update

royduin committed:
Added Laravel 5.5 package auto discovery support

oriceon committed:
Merge pull request #4 from royduin/master

Added Laravel 5.5 package auto discovery support

Commit:
fix "./composer.json" does not contain valid JSON

Commit:
escape single quote for the message

oriceon committed:
Merge pull request #5 from yoeunes/master

fix composer.json and escape single in render function

trollderiu committed:
Allow Toastr usage without 'toastr.options' config definition

oriceon committed:
Merge pull request #6 from trollderius/patch-2

Allow Toastr usage without 'toastr.options' config definition
